src/main/java/org/broadinstitute/hellbender/tools/validation/CompareReads.java

```diff
@@ -0,0 +1,129 @@
+package org.broadinstitute.hellbender.tools.validation;
+
+import htsjdk.samtools.SAMFlag;
+import htsjdk.samtools.SAMRecord;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.engine.ReadsDataSource;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import picard.cmdline.programgroups.DiagnosticsAndQCProgramGroup;
+
+import java.util.*;
+
+@DocumentedFeature
+@CommandLineProgramProperties(
+        summary = "Compares the base qualities of two SAM/BAM/CRAM files. The reads in the two files must have " +
+                "exactly the same names and appear in the same order.",
+        oneLineSummary = "Compares the base qualities of two SAM/BAM/CRAM files",
+        programGroup = DiagnosticsAndQCProgramGroup.class
+)
+public class CompareReads extends GATKTool {
+    @Argument(doc = "If output is given, the tool will return a bam with all the mismatching duplicate groups in the first specified file",
+            shortName = "I1", fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME, optional = true)
+    protected String input1;
+
+    @Argument(doc = "If output is given, the tool will return a bam with all the mismatching duplicate groups in the second specified input file",
+            shortName = "I2", fullName = "output2", optional = true)
+    protected String input2;
+
+
+    @Override
+    public void traverse() {
+        List<String> errorMessages = new ArrayList<>();
+
+        try(        ReadsDataSource reads1 = new ReadsDataSource(IOUtils.getPath(input1));
```

Comment:
the spacing here is really weird

Reply:
Whoops, I forgot to delete this tool from the branch... Uhh... I can include it if need be and clean it up